ktor-network/jvm/src/io/ktor/network/sockets/NIOSocket.kt

```diff
@@ -104,7 +105,7 @@ internal abstract class NIOSocketImpl<out S>(
         return try {
             channel.close()
             super.close()
-            socketContext.complete()
+//            socketContext.complete()
```

Comment:
Consider avoiding such comments

Reply:
Yep, I just left it here because we need to think a bit more about exception handling in this case. I removed the comment, but let's take a look at this behavior closely one more time.